### DISCUSSION TITLE
Wails + Whatsmeow

### DISCUSSION THREAD

**Discussion by bayaderpack** [tulir/whatsmeow#564] [General]
I trying like 5 hours to connect these 2 but for some reason it dont work. I'm saving image of qr code using this library github.com/skip2/go-qrcode but for some reason it is not possible to connect the phone.

**Comment by azickri** [comment]
What obstacles did you experience, more precisely?

**Reply by bayaderpack** [reply]
When I connect them I cant login with QR code at least I dont know how to make it work

**Reply by azickri** [reply]
At least, provide the code you created

**Reply by bayaderpack** [reply]
https://github.com/bayaderpack/whatsapp-bot

**Reply by azickri** [reply]
https://github.com/bayaderpack/whatsapp-bot/blob/main/whatsapp.go#L45

Can you check in this line?

**Reply by bayaderpack** [reply]
The issue is in my app I click button connect and QR image is generated. I scan QR image and its start loading for 5-10 sec.
I get the message "Couldn't log in. Check your phone's internet connection and scan the QR code again".
In log I get this 
```
10:39:04.844 [Main INFO] Debug%!(EXTRA bool=true)
10:39:31.854 [Main INFO] Successfully paired MYJID
10:39:31.854 [Main INFO] Debug%!(EXTRA bool=false)
Login event: success
```

**Reply by azickri** [reply]
Maybe you can try with another device and number

**Reply by bayaderpack** [reply]
If I do it without wails part it connect normally
